### PR TITLE
Last minute bug fixes and tuning autonomous values

### PR DESCRIPTION
**Pull request by leviatronix**
while some of this code is not thoroughly tested or is known to be broken, i believe the quality of code is good or the problem can be easily fixed at next opportunity.

### CODE REVIEW

File: tiredrive/strategies.py [hunk 1]
```diff
@@ -76,12 +76,45 @@ def turn_forward_left(self):
...
+            self.robot.robotdrive.tankDrive(left_wheel, right_wheel)
+            yield
+
+    def wait(self):
+        while True:
+            self.robot.forward(0)
+            yield
 
 
 class Auto3StraightStrategy:
```
Comment (bb010g) [inline tiredrive/strategies.py]: Perhaps rename to ``Auto3ToteStrategy``?

File: tiredrive/strategies.py [hunk 2]
```diff
@@ -137,30 +168,32 @@ def maintain_winch(self):
...
-        while robot.right_encoder.get() <= revs0 + 60:
-            robot.forward(0.5)
+        while not robot.right_claw_whisker() and not robot.left_claw_whisker():
+            robot.forward(0.7)
             yield
 
     def drop_tote(self, i):
         robot = self.robot
         robot.winch_setpoint = robot.winch_setpoint_zero
+        durp = False
```
Comment (bb010g) [inline tiredrive/strategies.py]: Is ``durp`` a placeholder for a future condition? If so, why's it here now?

Comment (ariovistus) [inline tiredrive/strategies.py]: no, its just a flag to ensure the if statement three lines down is only run once.

File: tiredrive/strategies.py [hunk 3]
```diff
@@ -76,12 +76,45 @@ def turn_forward_left(self):
...
+        self.auto = ParallelGenerators()
+        self.auto.add("turn", self.turn(90))
+        self.auto.add("brake", self.brake(), after="turn")
+        self.auto.add("wait", self.wait(), after="brake")
+
+    def autonomousPeriodic(self):
+        self.auto.next()
+
+    def turn(self, angle):
+        while abs(self.robot.gyro.getAngle()) % 360 < angle:
```
Comment (ariovistus) [inline tiredrive/strategies.py]: wouldn't you be wanting to compare the <longthing>difference between the initial gyro value and the current gyro value</longthing> against angle?

File: tiredrive/strategies.py (hunk 2 above)
Comment (bb010g) [inline tiredrive/strategies.py]: Ah. Perhaps a more descriptive name could be used?

File: tiredrive/strategies.py (hunk 3 above)
Comment (leviatronix) [inline tiredrive/strategies.py]: yes. the code should be fixed so that it doesn't assume initial angle is 0

File: tiredrive/strategies.py (hunk 1 above)
Comment (ariovistus) [inline tiredrive/strategies.py]: addressed in #6

File: tiredrive/strategies.py (hunk 2 above)
Comment (ariovistus) [inline tiredrive/strategies.py]: addressed in #6

**Comment by bb010g** [comment]
Merged as part of #6.

**Pull request merged**